test/unit/service_test.rb

```diff
@@ -146,7 +146,7 @@ def test_update_account_default_service
 
   test 'service_token' do
     account = FactoryBot.create(:simple_account)
-    service = Service.new { |s| s.account_id = account.id; s.system_name = 'foo' }
+    service = Service.new { |s| s.account = FactoryBot.create(:simple_account); s.system_name = 'foo' }
```

Comment:
I do not understand what is the previous account created for then (the previous line), or at least I don't see why we save it in a variable that we never use.

Reply:
I should have removed it
```suggestion
    service = Service.new { |s| s.account = account; s.system_name = 'foo' }
```